### ISSUE TITLE
HP DL360 not returning disk metrics

### ISSUE THREAD

**Issue by brad-marshall** [jenningsloy318/redfish_exporter#59]
I've been testing this exporter against some Dell r640 and HP DL360s, and it seems to be pretty good.

Unfortunately the HP DL360s don't return any information for at least drive data, since it seems to be under a different redfish URL:

  /redfish/v1/Systems/1/SmartStorage/ArrayControllers/0/DiskDrives/0

but the Dell hardware seems to be under:

  /redfish/v1/Systems/System.Embedded.1/SimpleStorage/Controllers/RAID.Integrated.1-1

I can definitely see information like memory for the HPs, so it seems like its just storage related.

What further information can I provide to figure out what is going on here?

**Comment by jenningsloy318** [comment]
Hi @brad-marshall  
this exporter is levaraging upstream https://github.com/stmcginnis/gofish for dealing with the infomation extration, and maybe your device is not standard redfsih-compliant.

**Comment by brad-marshall** [comment]
Right, I'll do a double check to see if I have the latest firmware for the ilo and then see if it behaves differently, and file a bug on that library if it isn't.  Thanks!
